.gitignore

```diff
@@ -9,3 +9,6 @@ pom.xml
 /lib/
 /target/
 .lein-deps-sum
+.portal
+.lsp/
+.clj-kondo/
```

Comment:
You may want to add a `config.edn` in `.clj-kondo` at some point and then change this to `.cache` which ignores both `.lsp/.cache` and `.clj-kondo/.cache`.

Reply:
done.